datafusion/physical-expr/src/lib.rs

```diff
@@ -44,9 +43,13 @@ pub mod udf;
 #[cfg(feature = "unicode_expressions")]
 pub mod unicode_expressions;
 pub mod utils;
-pub mod var_provider;
 pub mod window;
 
+// backwards compatibility
+pub mod execution_props {
+    pub use datafusion_expr::execution_props::ExecutionProps;
```

Comment:
How about var_provider here?

`pub use datafusion_expr::var_provider::{VarProvider, VarType};`

Reply:
Good call. Thank you

```suggestion
    pub use datafusion_expr::execution_props::ExecutionProps;
    pub use datafusion_expr::var_provider::{VarProvider, VarType};
```